src/database/schema/ChatSchema.ts

```diff
@@ -237,7 +237,7 @@ class ChatSchema implements BaseDbSchema {
       channelId,
       userId: json?.user?.id,
       message: (json?.text || json?.message) ?? '',
-      type: json?.type,
+      type: json?.message_type ?? json?.type,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is similar to the one on line 214, adding an additional fallback for the `type` property. It's a good change if there are cases where the type could be under `message_type` instead of `type`.
```diff
-      type: json?.type,
+      type: json?.message_type ?? json?.type,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->